__tests__/app.test.js

```diff
@@ -48,3 +46,30 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200 should respond with the article object for a valid article ID", () => {
+    return request(app)
+      .get("/api/articles/1")
```

Comment:
Because we are expecting a specific article back which we know the contents of - this test could be made more robust by expecting the correct values for that article here as well instead of just expecting the article to have the given keys (this technique is great for when we have multiple objects which have different values though)